recipes/maxbin2/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "maxbin" %}
+{% set version = "2.2.1" %}
+{% set md5 = "3748e8f65b5bf18f8bd341210724cc5c" %}
+
+build:
+  number: 0
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+source:
+  fn: MaxBin-{{ version }}.tar.gz
+  url: http://downloads.jbei.org/data/microbial_communities/MaxBin/getfile.php?MaxBin-{{ version }}.tar.gz
+  md5: {{ md5 }}
+requirements:
+  build:
+    - gcc # [linux]
+    - llvm # [osx]
+  run:
+    - libgcc # [linux]
+    - perl
+    - bowtie2
+    - fraggenescan >=1.30
+    - hmmer
+    - idba
+test:
+  commands:
+    - MaxBin > /dev/null
+about:
+  home: "http://downloads.jbei.org/data/microbial_communities/MaxBin/MaxBin.html"
+  license: "BSD 3-clause"
+  summary: "MaxBin is software for binning assembled metagenomic sequences based on an Expectation-Maximization algorithm."
+
+extra:
+  recipe-maintainers:
+    - colinbrislawn
+    - keuv-grvl
+  doi: 10.1186/2049-2618-2-26
```

Comment:
You could also add MaxBin 2.0 paper : [10.1093/bioinformatics/btv638](https://dx.doi.org/10.1093/bioinformatics/btv638)
``` yaml
doi:
  - 10.1186/2049-2618-2-26
  - 10.1093/bioinformatics/btv638
```